src/cogent3/app/align.py

```diff
@@ -200,6 +200,8 @@ def __init__(
         if callable(guide_tree):
             self._make_tree = guide_tree
             guide_tree = None  # callback takes precedence
+        elif guide_tree is None and self._moltype.label == "protein":
+            self._distance = "paralinear"
```

Comment:
I think this logic (and thus my original suggestion) is not right. The problem is this will have no effect on construction of `dist_calc` (currently line 205). So I now think the `elif` clause is incorrect

Instead, insert lines 203-204 before line 199 and change it to
```python
if guide_tree is None and self._moltype.label == "protein":
    distance = "paralinear"
```

change line 204 to `distance = "paralinear"`